docs/en/sql-reference/data-types/decimal.md

```diff
@@ -4,13 +4,15 @@ sidebar_position: 42
 sidebar_label: Decimal
 ---
 
-# Decimal(P, S), Decimal32(S), Decimal64(S), Decimal128(S), Decimal256(S)
+# Decimal, Decimal(P, S), Decimal32(S), Decimal64(S), Decimal128(S), Decimal256(S)
 
 Signed fixed-point numbers that keep precision during add, subtract and multiply operations. For division least significant digits are discarded (not rounded).
 
 ## Parameters
 
-- P - precision. Valid range: \[ 1 : 76 \]. Determines how many decimal digits number can have (including fraction).
+Decimal(P) is equivalent to Decimal(P, 0). Similarly, the syntax Decimal is equivalent to Decimal(P, 0).
```

Comment:
The 2nd sentence is confusing, it is not immediately clear what P represents. Maybe: "Similarly, the syntax Decimal is equivalent to Decimal(10, 0)."? (this will make even more sense if both sentences were moved down to l. 17.)